src/EA.Weee.Web/Areas/Scheme/ViewModels/ReviewTransferNoteViewModel.cs

```diff
@@ -18,7 +20,9 @@ public class ReviewTransferNoteViewModel : RadioButtonStringCollectionViewModel,
 
         public ViewTransferNoteViewModel ViewTransferNoteViewModel { get; set; }
 
-        [Required(ErrorMessage = "Choose whether to approve, reject or return the evidence note transfer")]
+        public string RedirectTab => ManageEvidenceNotesDisplayOptions.ReviewSubmittedEvidence.ToDisplayString();
+
+        [Required(ErrorMessage = "Select whether you want to approve, reject or return the evidence note transfer")]
```

Comment:
Can we add a unit tests for ReviewTransferNoteViewModel to ensure that the properties have the correct Required attribute error messages and same for DisplayName. As looks like no tests files were changed for this so probably lacking in tests.